src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('animals` human age should be 0 if their own age < 15', () => {
+    expect(getHumanAge(0, 14)).toEqual([0, 0]);
+  });
+
+  test('animals` human age should be 1 if their own age > 14 and < 24', () => {
+    expect(getHumanAge(15, 23)).toEqual([1, 1]);
+  });
```

Comment:
The test description and expectation are consistent for ages between 15 and 23. Verify that the `getHumanAge` function implements this logic correctly.

Reply:
done